### PR TITLE
dsns 115

### PR DESCRIPTION
**Pull request by gordonyeong**
DSNS-115:
    1. Updated the path for the configuration file for production environment for correctness
    2. Updated the umask permission for consistency between prod
    and development

### CODE REVIEW

File: dags/nci_ard/nci_fetch_brdf.py [hunk 1]
```diff
@@ -57,25 +59,19 @@
...
     schedule_interval = None
 
-    params["umask"] = " umask 0017 "
+    landsat_repo_path_base_path = "/g/data/v10/work/landsat_downloads/landsat-downloader/USGS_GAC3_ancillary_processing/fetch"
 
     params[
         "config_file"
-    ] = "/home/547/lpgs/USGS_GAC3_ancillary_processing/fetch/airflow-config.yaml"  # TODO - durign prod deployment, please make sure you delpy the latest version of airflow-config.yaml from the landsat downloader repo
-
+    ] = "/home/547/lpgs/USGS_GAC3_ancillary_processing/fetch/airflow-config.yaml"
```
Comment (dunkgray) [inline dags/nci_ard/nci_fetch_brdf.py:66]:
```suggestion
    ] = "/g/data/v10/work/landsat_downloads/landsat-downloader/USGS_GAC3_ancillary_processing/fetch/airflow-config.yaml"
```

File: dags/nci_ard/nci_fetch_brdf.py [hunk 2]
```diff
@@ -57,25 +59,19 @@
     # todo = remove this line when it's ready to for review
     schedule_interval = None
 
-    params["umask"] = " umask 0017 "
+    landsat_repo_path_base_path = "/g/data/v10/work/landsat_downloads/landsat-downloader/USGS_GAC3_ancillary_processing/fetch"
```
Comment (dunkgray) [inline dags/nci_ard/nci_fetch_brdf.py:62]: landsat_repo_path_base_path - is it being used? Use or remove

**Review by dunkgray — APPROVED** [review]
That looks good.

**Review by dunkgray — APPROVED** [review]

File: dags/nci_ard/nci_fetch_brdf.py (hunk 2 above)
Comment (gordonyeong) [reply dags/nci_ard/nci_fetch_brdf.py:62]: taken care of. I did fix this but not sure what happeend to the commit. Sorry

**Pull request merged**